### PR TITLE
Adding MAROE Prefix verification.

### PR DESCRIPTION
**Pull request by Darshini-Parikh93**
- MAROE Prefix verification added for msg/32 and msg/64 for
EPID devices.
- Added verifyMaroePrefix function in cryptoService class.

Signed-off-by: Darshini Parikh <darshini.k.parikh@intel.com>

### CODE REVIEW

File: protocol/src/main/java/org/fido/iot/protocol/CryptoService.java [hunk 1]
```diff
@@ -77,6 +77,7 @@
 
   private static final BouncyCastleProvider BCPROV = new BouncyCastleProvider();
   private final SecureRandom secureRandom;
+  public static final String[] maroePrefixList = {"e2cb06bd11a323419d1d3563b809bc46"};
```
Comment (prateekcr) [inline protocol/src/main/java/org/fido/iot/protocol/CryptoService.java:80]: Can we move this to Const.java?

Comment (Darshini-Parikh93) [reply protocol/src/main/java/org/fido/iot/protocol/CryptoService.java]: Done

**Review by johneasterday — APPROVED** [review]
I assume the prefix value was verified? I know it differs slightly from the actual applet appId due to some mapping in the HostApp.

**Comment by Darshini-Parikh93** [comment]
> I assume the prefix value was verified? I know it differs slightly from the actual applet appId due to some mapping in the HostApp.

The prefix value was taken from the JSON based implementation. I don't have the prefix for the CBOR based implementation. If you have the value, I can update the PR.

**Review by prateekcr — APPROVED** [review]

**Pull request merged**